homeassistant/components/homewizard/coordinator.py

```diff
@@ -45,6 +44,19 @@ async def _async_update_data(self) -> DeviceResponseEntry:
             raise UpdateFailed("Device did not respond as expected") from ex
 
         except DisabledError as ex:
+            async_create_issue(
+                self.hass,
+                DOMAIN,
+                f"api_disabled_{self.config_entry.entry_id if self.config_entry is not None else ''}",
+                data={CONF_IP_ADDRESS: self.api.host, CONF_DEVICE_NAME: self.name},
+                is_fixable=True,
+                learn_more_url="https://www.home-assistant.io/integrations/homewizard/#enable-the-api",
+                severity=IssueSeverity.ERROR,
+                translation_key="api_disabled",
+                translation_placeholders={
+                    "device_name": self.name,
+                },
+            )
```

Comment:
Every time the coordinator triggers the `async_create_issue()` line, the repair panel is closed in the frontend. I think I do not need to call that function every time, is there a better way to do this?

I think the coordinator can just stop, but that brings the problem that this issue it not repaired by itself.